src/cli/new.rs

```diff
@@ -8,57 +8,29 @@ use cli::test::{
     TEST_DIR_NAME,
     DIST_DIR_NAME,
 };
-use util;
 use error::DefaultResult;
 use serde_json;
 use std::{
-    fs::{self, File},
+    fs::{self, File, OpenOptions},
     path::{PathBuf},
     io::Write,
 };
 
-struct TestRepo {
-    name: String,
-    url: String,
-    commit: Option<String>
+fn create_test_file(test_folder_path: &PathBuf, test_file_name: &str, test_file_contents: &str) -> DefaultResult<()> {
+    let dest_filepath = test_folder_path.join(test_file_name);
+    let mut file = OpenOptions::new().write(true).create(true).open(dest_filepath)?;
+    file.write_all(test_file_contents.as_bytes())?;
+    Ok(())
 }
 
-fn setup_test_folder(path: &PathBuf, test_folder: &str, test_repo: TestRepo) -> DefaultResult<()> {
-
-    // check if there was a specific commit to checkout, or just latest
-    match test_repo.commit {
-        Some(sha) => {
-            // clone the repo with history
-            util::run_cmd(path.clone(), "git".to_string(), vec![
-                "clone".to_string(),
-                "--quiet".to_string(),
-                test_repo.url.to_string(),
-                test_folder.to_string(),
-            ])?;
-            // checkout the desired commit
-            let temp_dir = path.join(test_repo.name.to_string());
-            util::run_cmd(temp_dir, "git".to_string(), vec![
-                "checkout".to_string(),
-                sha.to_string(),
-                "--quiet".to_string(),
-            ])?;
-        },
-        None => {
-            // clone the repo without history
-            util::run_cmd(path.clone(), "git".to_string(), vec![
-                "clone".to_string(),
-                "--quiet".to_string(),
-                "--depth".to_string(),
-                "1".to_string(),
-                test_repo.url.to_string(),
-                test_folder.to_string(),
-            ])?;
-        },
-    }
-
-    // remove the .git folder
-    fs::remove_dir_all(path.join(test_folder).join(".git"))?;
-
+fn setup_test_folder(path: &PathBuf, test_folder: &str) -> DefaultResult<()> {
+    let tests_path = path.join(test_folder);
+    fs::create_dir_all(tests_path.clone())?;
+    create_test_file(&tests_path, "index.js", include_str!("js-tests-scaffold/index.js"))?;
+    create_test_file(&tests_path, "package-lock.json", include_str!("js-tests-scaffold/package-lock.json"))?;
+    create_test_file(&tests_path, "package.json", include_str!("js-tests-scaffold/package.json"))?;
+    create_test_file(&tests_path, "README.md", include_str!("js-tests-scaffold/README.md"))?;
+    create_test_file(&tests_path, "webpack.config.js", include_str!("js-tests-scaffold/webpack.config.js"))?;
```

Comment:
include_str! has to be called with a string literal passed in, which is why the function signature/calls look this way, with the filename duplicated.